StyleSheets/StyleSheets/TextStyleSheet.swift

```diff
@@ -0,0 +1,56 @@
+import UIKit
+
+public protocol TextViewProtocol: AnyObject {
```

Comment:
You could say `where Self: UIView` as part of the protocol.

Reply:
Not sure exactly why but I'm getting this strange errors 😕

<img width="1620" alt="image" src="https://user-images.githubusercontent.com/1098602/49581762-e9310480-f94a-11e8-9a1f-2098103bac02.png">

Reply:
This is because it does not know it's a reference type, you either need to make a method `mutating` or make a protocol `AnyObject where Self: TextViewProtocol`

Reply:
But it's already defined in the definition of `TextViewConfiguratorProtocol`.

```swift
protocol TextViewConfiguratorProtocol: AnyObject
```